ee/federated-saml/api/admin/index.ts

```diff
@@ -35,12 +37,13 @@ const handlePOST = async (req: NextApiRequest, res: NextApiResponse) => {
 const handleGET = async (req: NextApiRequest, res: NextApiResponse) => {
   const { samlFederatedController } = await jackson();
 
-  const { offset, limit, pageToken } = req.query as { offset: string; limit: string; pageToken?: string };
+  const { pageOffset, pageLimit, pageToken } = parsePaginateApiParams(req.query as PaginateApiParams);
```

Comment:
@niwsa `as PaginateApiParams` looks unnecessary. Adjust `parsePaginateApiParams` method to receive query parameters (type of req.query?) as an object instead.